gnucash/report/reports/support/taxinvoice.eguile.scm

```diff
@@ -331,7 +331,7 @@
         <td align="right"><?scm:d (fmtnumeric qty) ?></td>
       <?scm )) ?>
       <?scm (if (or units? qty? discount?) (begin ?>
-        <td align="right"><?scm:d (fmtmoney currency each) ?></td>
+        <td align="right"><?scm:d (fmtprice currency each) ?></td>
```

Comment:
There's already `gnc:default-price-renderer`:

```suggestion
        <td align="right"><?scm:d (gnc:default-price-renderer currency each) ?></td>
```

Reply:
OK. Using that suggests that I should also replace `fmtmoney` with `gnc:default-html-monetary-renderer` and maybe do the same for `gnc:monetary->string`.

Reply:
> `fmtmoney`…

Nah. `gnc:default-html-monetary-renderer` seems excessively convoluted.